static/js/quality-chooser.js

```diff
@@ -44,7 +44,22 @@ $(document).ready(function() {
                   '&allowed=' + selectedAllowed +
                   '&preferred=' + selectedPreffered
         $.getJSON(url, function(data) {
-            $('#backlogged_episodes').text('Currently you have ' + data[1] + ' backlogged episodes. With this change you would have ' + data[0] + ' backlogged episodes');
+            var new_backlogged = parseInt(data[0])
+            var current_backlogged = parseInt(data[1])
+            var variation = Math.abs(new_backlogged - current_backlogged)
```

Comment:
Also not sure why you're using Math.abs when it'll strip negative numbers, etc. just use `-`.

```
Math.abs('-1');     // 1
Math.abs(-2);       // 2
Math.abs(null);     // 0
Math.abs("string"); // NaN
Math.abs();         // NaN
```